### PR TITLE
Support latest proposals

### PR DESCRIPTION
**Pull request by novemberborn**
See commits.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,9 +5,6 @@
   "license": "MIT",
   "repository": "avajs/babel-preset-stage-4",
   "author": "Mark Wubben (https://novemberborn.net)",
-  "publishConfig": {
-    "access": "public"
-  },
```
Comment (sindresorhus) [inline package.json:10]: Why was this removed? https://github.com/sindresorhus/np#public-scoped-packages

Comment (novemberborn) [reply package.json:10]: That's only necessary when the package is first published. Afterwards npm knows it's public.

Comment (sindresorhus) [reply package.json:10]: Oh, I never realized. I've been putting this config in many packages.
